### DISCUSSION TITLE
Custom Config File Location

### DISCUSSION THREAD

**Discussion by 0xdade** [coddingtonbear/obsidian-web#177] [Ideas]
I'm not sure if there's a limitation of the Chrome extension APIs that would prevent this or not, but I'd love to be able to put my Obsidian Web authoritative config file directly into a vault and have it load from there / sync to there when I change settings. This way my Obsidian Web templates / actions are saved into my vault and can be reloaded into another browser anywhere else that I have Obsidian running.

I know Chrome has built in sync support to your google account, which is also okay, but I have many google accounts and want to sync my Obsidian Web behavior across all of them, basically. So the google account sync alone isn't sufficient.

If it's not possible with the Chrome extension APIs, that's fine too. Perhaps another option would be a setting in Obsidian Web that lets me specify a place in my vault for the extension to automatically write the config file whenever there are changes -- this could be done using the built in integration with the Rest API plugin.

For example, I am currently manually saving an export of my config into `Meta/Plugins/Obsidian Web/` every time I make a change.

**Comment by coddingtonbear** [comment]
I think you might be interested in this discussion: #114 -- this idea has been raised before.

**Reply by 0xdade** [reply]
Thanks Adam, I swear I looked for references to config before submitting this, but somehow missed that one!
